src/classes/issues-processor.ts

```diff
@@ -911,6 +922,48 @@ export class IssuesProcessor {
     return this.options.removeStaleWhenUpdated;
   }
 
+  private async _removeLabelsWhenUpdatedFromStale(
+    issue: Issue,
+    labels: Readonly<string>[]
+  ): Promise<void> {
+    if (!labels.length) {
+      return;
+    }
+
+    const issueLogger: IssueLogger = new IssueLogger(issue);
+
+    issueLogger.info(
+      `Removing any labels specified via remove-labels-when-updated-from-stale...`
```

Comment:
There is a better way actually to mention an option (to have a generated link in the logs at the end)
```suggestion
      `Removing all the labels specified via the ${this._logger.createOptionLink(
            Option.RemoveLabelsWhenUpdatedFromStale
          )} option...`
```
You need to update the Option enum as well